### PR TITLE
add web pages

### PR DESCRIPTION
**Pull request by erezdav22**
Adding web pages for ultimatjob site as requested on the issues we opened.
This is how you navigate in the html pages:
1.  ultimate job- home page
2.  ultimate job- sign up
3. personal area (here the user will be able to search for his jobs)
4. available jobs (here the jobs that fit the user will appear)

also, there is a sign in page for users who already have an account- after that page they will be navigated to page 3.

**This PR will solve the issues above:**
#41 #42 #43 #44 #45

### CODE REVIEW

**Comment by BarBenNaim** [comment]
I think you upload also beyond tutorial files

**Comment by erezdav22** [comment]
> I think you upload also beyond tutorial files

yes by mistake, i deleted them from the PR now, thanks

File: ultimatejobweb/migrations/0001_initial.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('profession', models.CharField(max_length=128)),
-                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
+                ('user', models.OneToOneField(on_delete=django.db.models.deletion.CASCADE,to=settings.AUTH_USER_MODEL)),
```
Comment (simzacks) [inline ultimatejobweb/migrations/0001_initial.py]: why did you change this file? It is auto-generated and should be changed

**Review by simzacks — CHANGES_REQUESTED** [review]
Please remove spaces from file names, make all filenames lowercase and don't use "-", you can use "_" if you want a space between the names in  file.

File: ultimatejobweb/HomeCSS.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
+* {
```
Comment (Yarboa) [inline ultimatejobweb/HomeCSS.css]: Is it a comment? can you remove it please?

Comment (erezdav22) [reply ultimatejobweb/HomeCSS.css]: Removed.

File: ultimatejobweb/migrations/0001_initial.py (hunk 1 above)
Comment (erezdav22) [reply ultimatejobweb/migrations/0001_initial.py]: I just changed all files names as you requested.
Fixed the auto-generated Python file, too, and now it has no problems with flake8. (I changed it at first because flake8 showed there was too many characters in line 30, but now it is working well.
Is the PR approved? @simzacks

**Comment by Yarboa** [comment]
Nice work Erez
